### ISSUE TITLE
Genesis "Game Series Collections" items pointing to old versions

### ISSUE THREAD

**Issue by eguevaralopez** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#407]
The following ROMs can be replaced for newer versions already in the SMDB.

The format is the following;
```
Reported missing ROM after building pack
Line in SMDB to replace
Updated line with newer version of ROM
```

Relevant lines are;
```
004 Street Fighter II SE (U) Pyron Color Restore v6.0.md	52f9ec0f0ed087b238a7664b6f2af7283ec99af1d0dee835c9060180927ea819
52f9ec0f0ed087b238a7664b6f2af7283ec99af1d0dee835c9060180927ea819	Mega EverDrive/4 Game Series Collections/Best-Of Lists/Top 100 List - Retro-Sanctuary.com/004 Street Fighter II SE (U) Pyron Color Restore v6.0.md	b0e2dbae3bbcd41956c8e0edf6e59cf5f4e3f808	8fdce3ca6cb7385f8006eb3419de5b81	c582a63c
80ae49883ea5d59270873fcecaa2aed771b0458af8cf50c65540ba59b7ac869b	Mega EverDrive/4 Game Series Collections/Best-Of Lists/Top 100 List - Retro-Sanctuary.com/004 Street Fighter II SCE Pyron Arcade Restore+Z80 Sound (Hack) v7 Pyron, Stef Z80 v7.md	8c19521a9e864e710d0464ab98875dd67a5f3885	664d3e384930b5b9a76898cb21051ad1	293d2cb1
```

```
Altered Beast - Voice+Color (W) Pyron Alt Style v2, Linkuei Voice v1.md	023b1d27728fa6df8035b36489943352d1ffe7c54ad2dd60f418372a1aeaf25d
023b1d27728fa6df8035b36489943352d1ffe7c54ad2dd60f418372a1aeaf25d	Mega EverDrive/4 Game Series Collections/Miscellaneous/Halloween Time/Altered Beast - Voice+Color (W) Pyron Alt Style v2, Linkuei Voice v1.md	83ec2d377600fb267bf79e1f67f02cee22109c36	019353b9bb43492885ed3597c5db2bb1	499c9696
REMOVE, UPDATED LINE IS ALREADY IN PLACE
```

```
Splatterhouse 2 (U) Arcade Colors - Classic Rick v1 fusaru.md	09e357103bc2760df03f83034f4ea0c5c92732dc3f02dd6c8ce7ece57e601de8
09e357103bc2760df03f83034f4ea0c5c92732dc3f02dd6c8ce7ece57e601de8	Mega EverDrive/4 Game Series Collections/Miscellaneous/Halloween Time/Splatterhouse 2 (U) Arcade Colors - Classic Rick v1 fusaru.md	3b74cce10c13633c16d08fef4421193cbc9857c4	6a578bec495d6d69a1679d3058042fb2	f31fe3f7
1317e8631cf9f22cdefee0b9b85b7cd8f49e33d69894e2c3bbfa3410c4c70a21	Mega EverDrive/4 Game Series Collections/Miscellaneous/Halloween Time/Splatterhouse 2 Arcade Colors - Classic Rick (Hack) v1 fusaru.md	ec49d9ace538d471e946a84c27320103f9d05e82	4285b26465a295976be03c348eb77c62	1ea2a0c9
```

```
Splatterhouse 2 (U) Arcade Colors - Skull Mask Rick v1 fusaru.md	c54e04a70e698e7c28b9814a0b9230a550774698c93eefd59c746f4c3d9c9f10
c54e04a70e698e7c28b9814a0b9230a550774698c93eefd59c746f4c3d9c9f10	Mega EverDrive/4 Game Series Collections/Miscellaneous/Halloween Time/Splatterhouse 2 (U) Arcade Colors - Skull Mask Rick v1 fusaru.md	59b741f30fa2ca342f4485186017442e8413ca38	b33a2fab7a73b9f2959d73fceefa503b	564ef19e
0881b90ee461188b80b27e6b9ae5601a4ae783f5511e4599e0171ff6b428b745	Mega EverDrive/4 Game Series Collections/Miscellaneous/Halloween Time/Splatterhouse 2 Arcade Colors - Skull Mask Rick (Hack) v1 fusaru.md	ce3e5244bd973f96a63f3b5f79a9b9054624b0d9	927bcc973950b1bdf7a9fee4b06b21f4	291211ec
```

```
Miniplanets (PD) 1-26-2017 Sik.md	67b7e4d2bcac056ddadf195a5792df1eebdfa2173da87cd08a039931094decd5
67b7e4d2bcac056ddadf195a5792df1eebdfa2173da87cd08a039931094decd5	Mega EverDrive/4 Game Series Collections/Best-Of Lists/V's Recommended Games - 4chan/Miniplanets (PD) 1-26-2017 Sik.md	5da2996cacf78c12776cd1915bdf6935efa03e03	3ec9c76756fd0cd2cde792ce5ae33afd	48460e4a
33688a957c5c857730adde492c52d3095113f82e304f36f63761ef3373b44d73	Mega EverDrive/4 Game Series Collections/Best-Of Lists/V's Recommended Games - 4chan/Miniplanets (PD) 7-2018 Sik.md	88ccc6781746fb31c1e25b82568ad2a58071ea89	b9d6441165975395b689925ae95ec044	8ea40d2f
```

A question, while I'm here: why were the indies (e.g., Xeno Crisis, Ultracore, etc) purged? I tried looking for clues through the history but they were just removed on April 10th and I can't find the reason why in any note.

Thanks for the hard work!

**Comment by frederic-mahe** [comment]
thanks @eguevaralopez 

We would like to see more people develop and sell new games for old consoles. However, we've decided to exclude games from indie studios from our SMDB files (except public demos and open-source games). The reason is that you can find ROM packs on the web that are directly derived from our SMDB files. Excluding indie games is our way to try to protect and help indie studios, at least a little bit.

**Comment by frederic-mahe** [comment]
fixed in 0364d19cab0c0a605e485a5c9cbfc76fcf77d0aa
